### PR TITLE
Feature/tech 108 optimise release action

### PR DESCRIPTION
**Pull request by sara-wokounova**

### CODE REVIEW

**Review by ceyhunhan — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -78,11 +84,20 @@ jobs:
...
         if: inputs.docker_image == true
         run: ./gradlew jib -PgithubRun
       - name: 'Build and publish jar'
         if: inputs.docker_image == false
         run: ./gradlew publish
       - name: 'Push release'
-        run: ./gradlew pushRelease
\ No newline at end of file
+        run: ./gradlew pushRelease
+      - name: 'Create summary'
```
Comment (Jakubaakk) [inline .github/workflows/release.yml:101]: Maybe add
```
        if: inputs.docker_image == true
 ```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -78,11 +84,20 @@ jobs:
           path: /home/runner/work/${{ env.GITHUB_REPO_NAME }}/${{ env.GITHUB_REPO_NAME }}/build/reports/tests/test
       - name: 'Create release'
         run: ./gradlew createRelease -Prelease.forceVersion=${{github.event.inputs.force_version}}
+      - name: 'Get version'
+        id: get-version
+        run: |
+          VERSION=$(./gradlew cV -q -Prelease.quiet)
+          echo "Pushing version v$VERSION to ${{ inputs.aws_region }}"
```
Comment (Jakubaakk) [inline .github/workflows/release.yml:91]: This action is used for the components as well and they are missing `inputs.aws_region`

File: .github/workflows/release.yml (hunk 1 above)
Comment (sara-wokounova) [reply .github/workflows/release.yml:101]: Added, thanks

File: .github/workflows/release.yml (hunk 2 above)
Comment (sara-wokounova) [reply .github/workflows/release.yml:91]: Changed the wording to not contain `inputs.aws_region`, thanks.

**Pull request merged**